src/headless/converse-chatboxes.js

```diff
@@ -747,7 +747,7 @@ converse.plugins.add('converse-chatboxes', {
                     'references': this.getReferencesFromStanza(stanza),
                     'subject': _.propertyOf(stanza.querySelector('subject'))('textContent'),
                     'thread': _.propertyOf(stanza.querySelector('thread'))('textContent'),
-                    'time': delay ? delay.getAttribute('stamp') : moment().format(),
+                    'time': delay ? moment(delay.getAttribute('stamp')).format() : moment().format(),
```

Comment:
Can you please explain again why this is necessary?

Reply:
As discussed on the chat, but for everyone else's reference: the explanation is in the commit message https://github.com/laszlovl/converse.js/commit/96da5c954b3c83ac6c7c74865f311923748b7976

Reply:
Wouldn’t it be better to keep the time/moment object in memory, rather than its string representation, for further manipulation from other parts of the code?

Reply:
You can only store string key/value pairs in localStorage.